live-editing/configs/DateTimeEditorConfigGenerator.ts

```diff
@@ -0,0 +1,28 @@
+import { IgxDateTimeEditorDirective, IgxInputGroupModule } from "igniteui-angular";
+
+import { DateТimeЕditorSample1Component } from "../../src/app/scheduling/datetimeeditor/datetimeeditor-sample-1/datetimeeditor-sample-1.component";
+
+import { Config } from "./core/Config";
+import { AppModuleConfig } from "./core/AppModuleConfig";
+export class DateTimeEditorConfigGenerator implements IConfigGenerator {
+    public generateConfigs(): Config[] {
+        const configs = new Array<Config>();
+
+        // date time editor sample 1
+        configs.push(
+            new Config({
+                component: DateТimeЕditorSample1Component,
+                appModuleConfig: new AppModuleConfig({
+                    imports: [
```

Comment:
add IgxInputGroupModule in the imports and ngImports
```suggestion
       imports: [IgxInputGroupModule, DateTimeBasicComponent, IgxDateTimeEditorModule],
          ngDeclarations: [DateTimeBasicComponent],
          ngImports: [IgxInputGroupModule, IgxDateTimeEditorModule]            
```